package.json

```diff
@@ -147,79 +148,66 @@
         "configurationAttributes": {
           "launch": {
             "required": [
-              "cwd",
-              "startupClass"
+              "mainClass"
             ],
             "properties": {
-              "cwd": {
+              "projectName": {
                 "type": "string",
-                "description": "Current working directory (Defaults to workspace root directory).",
-                "default": "${workspaceRoot}"
+                "description": "Name of the project that contains the main class (Optional, only required if the main class is contained in multiple projects)",
+                "default": ""
               },
-              "startupClass": {
+              "mainClass": {
                 "type": "string",
-                "description": "Startup class (Should be full qualified name e.g. org.eclipse.xxx.MainClass)",
+                "description": "The fully qualified name of the debuggee application's main class (e.g. org.eclipse.xxx.MainClass).",
                 "default": ""
               },
-              "classpath": {
+              "programArguments": {
                 "type": "string",
-                "default": "",
-                "description": "The classpath to be passed to jvm launcher (Optional, it would override the default classpath)"
+                "description": "The program arguments (Optional).",
+                "default": ""
               },
-              "sourcePath": {
+              "vmArguments": {
+                "type": "string",
+                "description": "The VM arguments (Optional).",
+                "default": ""
+              },
+              "classPath": {

```

Comment:
Do we need to use the plural form like "classPaths"?

Reply:
Both `java.exe` and eclipse use singular form for class path configuration, so i prefer to use singular form.